libraries/pico_graphics/pico_graphics.hpp

```diff
@@ -293,6 +293,8 @@ namespace pimoroni {
     virtual void frame_convert(PenType type, conversion_callback_func callback);
     virtual void sprite(void* data, const Point &sprite, const Point &dest, const int scale, const int transparent);
 
+    virtual bool render_pico_vector_tile(const Rect &bounds, uint8_t* alpha_data, uint32_t stride, uint8_t alpha_type) { return false; }

```

Comment:
Not sure if it would be preferable to expose the pretty_poly antialias type here instead of having magic numbers that match that enum?

Reply:
Have to be careful about too tightly coupling PicoGraphics and PicoVector since it's entirely possible to have C++ or MicroPython builds that include the former but not the latter. In this case perhaps it's PicoGraphics that should define the enum type for anti-aliasing so that PicoVector can inherit it.